Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -395,8 +395,73 @@ The key `flexform` is followed by the field which holds the Flexform data (`pi_f
 How to Access FlexForms From Fluid
 ----------------------------------
 
-Flexform settings can be read from within a Fluid template using
-:html:`{settings}`.
+If you are using an Extbase controller, FlexForm settings can be read from within a Fluid template using
+:html:`{settings}`. See the note on naming restrictions in :ref:`How to Read Flexforms From an Extbase Controller Action <_read-flexforms-extbase>`.
+
+If you defined your :typoscript:`FLUIDTEMPLATE` in TypoScript, you can assign single variables like that:
+
+.. code-block:: typoscript
+
+   my_content = FLUIDTEMPLATE
+   my_content {
+     variables {
+       categories = TEXT
+       categories.data = flexform: pi_flexform:categories
+     } 
+   }
+
+In order to have all FlexForm settings available, you can add a custom DataProcessor:
+
+.. code-block:: typoscript
+
+   my_content = FLUIDTEMPLATE
+   my_content {
+     dataProcessing {
+       10 = Your\Ext\DataProcessing\FlexFormProcessor
+     }
+   }
+
+.. code-block:: php
+   namespace Your\Ext\DataProcessing;
+   
+   use TYPO3\CMS\Core\Service\FlexFormService;
+   use TYPO3\CMS\Core\Utility\GeneralUtility;
+   use TYPO3\CMS\Frontend\ContentObject\DataProcessorInterface;
+   
+   class FlexFormProcessor implements DataProcessorInterface
+   {
+       /**
+        * @var FlexFormService
+        */
+       protected $flexFormService;
+       
+       public function __construct(FlexFormService $flexFormService) {
+           $this->flexFormService = $flexFormService;
+       }
+       
+       public function process(
+           ContentObjectRenderer $cObj,
+           array $contentObjectConfiguration,
+           array $processorConfiguration,
+           array $processedData
+       ): array {
+           $originalValue = $processedData['data']['pi_flexform'];
+           if (!is_string($originalValue)) {
+               return $processedData;
+           }
+   
+           $flexformData = $this->flexFormService->convertFlexFormContentToArray($originalValue);
+           $processedData['flexform'] = $flexformData;
+           return $processedData;
+       }
+   }
+
+.. code-block:: yaml
```

Comment:
Maybe link to https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ApiOverview/DependencyInjection/Index.html#configure-dependency-injection-in-extensions instead to avoid duplication and have an explanation?

Reply:
ok. Maybe the wording should be "... setup <link>Dependency Injection</link>, for example with an 'autowire' directive." since there are other possibilities as well?

Reply:
I've reduced this to an "see also" admonition now which requires no wording at all.